src/System.Diagnostics.Contracts/src/System.Diagnostics.Contracts.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <Platform Condition="'$(Platform)' == ''">AnyCPU</Platform>
+    <AssemblyName>System.Diagnostics.Contracts</AssemblyName>
+    <OutputType>Library</OutputType>
+    <ProjectGuid>{27328738-30D1-434a-8867-5C8ACACB91A8}</ProjectGuid>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
+    <NuGetTargetFrameworkMoniker>ASP.NetCore, version=v5.0</NuGetTargetFrameworkMoniker>    
+  </PropertyGroup>
```

Comment:
Where is your reference to mscorlib? I would still like to have the list of references (aka seed assemblies) explicitly add to the project so it is clear what the seeds are.